.github/workflows/lint.yaml

```
@@ -0,0 +1,17 @@
+name: Lint
+
+on: [push, pull_request, workflow_dispatch]
+
+permissions:
+  contents: read
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
```

`3.x` is "the latest stable version of Python", currently 3.11, and we won't need to update it when 3.12 is out.

https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#specifying-a-python-version

```suggestion
          python-version: "3.x"
```